.github/workflows/kgw-test-reuse.yaml

```diff
@@ -165,6 +166,46 @@ jobs:
           sed -i "s/\${UID}:\${GID}/${testUserID}:${testGroupID}/g" test/integration/docker-compose.override.yml
           KIT_LOG_LEVEL=warn go test -count=1 -timeout 0 ./test/integration -run ^TestKGW$ -v
 
+      - name: Run kwil-js test against kwild & kgw
+        run: |
+          cd deployments/compose/kwil
+          echo "run kwild in docker"
+          docker compose up -d
```

Comment:
Suprisingly this doesn't require the 'docker-compose.overwrite' to bypass the permission issue like we did in 'CI' workflow, but it works.